src/generated/java/com/prowidesoftware/swift/model/mt/MtSequenceInfoEnum.java

```diff
@@ -0,0 +1,131 @@
+/*
+* Copyright 2006-2024 Prowide
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package com.prowidesoftware.swift.model.mt;
+
+/**
+* Enumeration representing all possible sequence paths for each MT (Message Type) schema.
+* Each constant in this enumeration corresponds to a specific MT schema, identified by its
+* sequence paths as a comma-separated string (CSV) for SRU 2023.
+*
+* <p>For example, the MT508 message type may have sequence paths such as "A", "A/A1", "B", "B/B1", and "C",
+* which would be represented as the CSV string "A,A/A1,B,B/B1,C".
+*
```

Comment:
Add. "Internal loops not clearly identified and named in the standard as a sequence, are not included, for example ...."

Reply:
Ok